wrapper/src/test/java/integration/refactored/container/tests/ReadWriteSplittingTests.java

```diff
@@ -417,71 +463,244 @@ public void test_setReadOnlyFalse_allInstancesDown() throws SQLException {
   }
 
   @TestTemplate
+  @EnableOnTestFeature(TestEnvironmentFeatures.NETWORK_OUTAGES_ENABLED)
+  @DisableOnTestDriver(TestDriver.MARIADB) // TODO: investigate and fix
   // Tests use Aurora specific SQL to identify instance name
   @EnableOnDatabaseEngineDeployment(DatabaseEngineDeployment.AURORA)
-  public void test_readerLoadBalancing_autocommitTrue() throws SQLException {
-    try (final Connection conn = DriverManager.getConnection(getUrl(), this.propsWithLoadBalance)) {
+  public void test_transactionResolutionUnknown() throws SQLException {
```

Comment:
This test was meant for reader load balancing which is now removed, so we can remove this test too.